applications/MeshingApplication/tests/cpp_tests/test_meshing_utilities.cpp

```diff
@@ -28,7 +28,7 @@ namespace Kratos {
 
         void CreateDummy2DNoModelPartPropertiesModelPart(ModelPart& rModelPart)
         {
-            Properties::Pointer p_elem_prop = Kratos::make_shared<Properties>(0);
+            Properties::Pointer p_elem_prop(0);
```

Comment:
this is not the same is it?
this is now an uninitialized pointer right?

Reply:
But they are the same for intrusive pointer